### PR TITLE
doc: add doc for blacklisting older CephFS clients

### PR DESCRIPTION
**Pull request by batrick**
Fixes: http://tracker.ceph.com/issues/39130
Signed-off-by: Patrick Donnelly <pdonnell@redhat.com>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

File: doc/cephfs/administration.rst [hunk 1]
```diff
@@ -213,6 +213,28 @@ does not change a MDS; it manipulates the file system rank which has been
 marked damaged.
 
 
+Minimum Client Version
+----------------------
+
+It is sometimes desirable to set the minimum version of Ceph a client must be
```
Comment (baruza) [inline doc/cephfs/administration.rst]:
```suggestion
It is sometimes desirable to set the minimum version of Ceph that a client must be
```

File: doc/cephfs/administration.rst [hunk 2]
```diff
@@ -213,6 +213,28 @@ does not change a MDS; it manipulates the file system rank which has been
...
+
+    fs set <fs name> min_compat_client <release>
+
+For example, to only allow Nautilus clients, use:
+
+::
+
+    fs set cephfs min_compat_client nautilus
+
+Clients running an older version will automatically be evicted.
```
Comment (baruza) [inline doc/cephfs/administration.rst]:
```suggestion
Clients running an older version will be automatically evicted.
```

**Comment by baruza** [comment]
@batrick Patrick, I found just two nitpicks, otherwise looks good :)

**Comment by batrick** [comment]
Fixed, thanks @baruza

**Comment by baruza** [comment]
@batrick Patrick, I'm not familiar with the process here, should I mark the review as done somewhere? I'm confused by "at least one approving review is required" - is anything else required from me apart from the review?

**Comment by batrick** [comment]
You can go to "Files changed" and click the green drop down on the top right "Review changes". Then "approve".

You may need added to the Ceph org to be able to do that though. Not sure.

**Comment by batrick** [comment]
@liewegas can we add @baruza to the Ceph org? Maybe add a docs team too.

**Comment by liewegas** [comment]
> @liewegas can we add @baruza to the Ceph org? Maybe add a docs team too.

Already a member!

**Comment by baruza** [comment]
@batrick got it, thanks!

**Pull request merged**